openlibrary/catalog/marc/tests/test_data/bin_expect/ithaca_two_856u.json

```diff
@@ -1,6 +1,6 @@
 {
   "publishers": [
-    "[s.n."
+    "s.n."
```

Comment:
I think "[s.n.]" would be preferable since it's a comment, not an actual publisher. Ditto whatever Latin abbreviation is used for "no date"

Reply:
The three common abbreviations are [s.n.] sine nomine (without name), [s.l.] sine loco (without place), and [n.d.] no date. All three should be left in square brackets since they don't represent actual text from the cataloged item.

Reply:
Thanks, I agree with this. Resolved (at least for publisher info) in e8084193a895d8ee81200f49093389a3887479ce , which can be used as a basis to extend the abbreviation support. We'd need some new test cases for other names etc.